pages/collections/_type/_.vue

```diff
@@ -1,59 +1,65 @@
 <template>
-  <b-container
+  <div
     data-qa="entity page"
-    fluid
     class="entity-page"
   >
-    <b-row class="flex-md-row pt-5 bg-white mb-4">
-      <b-col
-        cols="12"
-      >
-        <b-container class="mb-5">
-          <EntityDetails
-            :description="description"
-            :is-editorial-description="hasEditorialDescription"
-            :title="title"
+    <b-container
+      fluid
+      class="p-0"
+    >
+      <b-row class="flex-md-row pt-5 bg-white mb-4">
+        <b-col
+          cols="12"
+        >
+          <b-container class="mb-5">
```

Comment:
Suggestion: perhaps add the classes "px-0 px-md-1" here.
`<b-container class="mb-5 px-0 px-md-1">`

Reply:
ok have tested and this works 'mb-5 px-0 px-md-3'